library/src/main/java/com/yayandroid/locationmanager/providers/locationprovider/GooglePlayServicesLocationProvider.java

```diff
@@ -150,7 +150,7 @@ public void onLocationChanged(Location location) {
         // Set waiting as false because we got at least one, even though we keep tracking user's location
         setWaiting(false);
 
-        if (!getConfiguration().keepTracking()) {
+        if (!getConfiguration().keepTracking() && getSourceProvider().isGoogleApiClientConnected()) {
```

Comment:
Hi @yayaa, Can you please tell me why I am getting this error "continuous-integration/travis-ci/pr — The Travis CI build failed". What I need to do resolve this. I found this issue in new pull request also. Thanks :)